service/collector/endpoint.go

```diff
@@ -107,6 +109,17 @@ func (e *Endpoint) Collect(ch chan<- prometheus.Metric) error {
 		}
 		clusterID := kvmConfig.Name // https://golang.org/doc/faq#closures_and_goroutines
 
+		appList, err := e.g8sClient.ApplicationV1alpha1().Apps(kvmConfig.Spec.Cluster.ID).List(ctx, v1.ListOptions{})
```

Comment:
I really wanted to directly get the app by name here but it turns out that clients can call the app whatever they want.